osvr/RenderKit/osvr_display_configuration.cpp

```diff
@@ -304,27 +304,24 @@ inline void parseDistortionMonoPointMeshes(
     }
 }
 
-inline void parseDistortionRGBPointMeshes(
-    Json::Value const& distortion,
-    osvr::renderkit::RGBPointDistortionMeshDescriptions& mesh) {
-    Json::Value myDistortion = distortion;
+inline void parseDistortionRGBPointMeshes(Json::Value myDistortion,
+                                          osvr::renderkit::RGBPointDistortionMeshDescriptions& mesh) {
     Json::Reader reader;
 
     // See if we have the name of an external file to parse.  If so, we open it
     // and grab its values to parse.  Otherwise, we parse the ones that they
     // sent in.
     auto externalDistortion =
-        getExternalDistortionFile("RGB point samples", reader, distortion,
-                                  "rgb_point_samples_external_file");
+        getExternalDistortionFile("RGB point samples", reader, myDistortion, "rgb_point_samples_external_file");
     if (externalDistortion) {
-        myDistortion = externalDistortion.distortion;
+        myDistortion.swap(externalDistortion.distortion);
```

Comment:
Probably want a comment here saying why we're calling swap when we don't intend to replace the externalDistortion value with myDistortion.